app/controllers/backend/outgoing_payment_modes_controller.rb

```diff
@@ -31,6 +31,7 @@ class OutgoingPaymentModesController < Backend::BaseController
       t.column :name
       t.column :cash, url: true
       t.column :with_accounting
+      # t.column :sepa
```

Comment:
If enable I get an `undefined method 'text' for "<div class=\"checkbox-true\"></div>":ActiveSupport::SafeBuffer`.
Is `t.column :sepa` the correct way to add a "boolean" column to the table ?